package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "prepare": "husky install",
     "build": "turbo run build",
-    "build-vscode": "turbo run build --filter=neo4j-for-vscode",
+    "build-vscode": "turbo run neo4j-for-vscode#dev",
```

Comment:
I couldn't spot anything production-y using this task.. so I changed it to use the new dev task

this dev task does _not_ pass a `--production` flag to our esbuild runner, thus it produces a sourcemap and does not minify the extension script

this allows us to debug the extension